data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/schema/ErrorListBadRequest.java

```diff
@@ -67,4 +72,31 @@ public int hashCode() {
   public String toString() {
     return "ErrorListBadRequest{" + "message='" + message + '\'' + ", errors=" + errors + '}';
   }
+
+  public static SerializableTypeDefinition<ErrorListBadRequest> getStacktraceTypeDefinition() {
+    return SerializableTypeDefinition.object(ErrorListBadRequest.class)
+        .name("StacktraceErrorListBadRequest")
+        .withField("code", INTEGER_TYPE, ErrorListBadRequest::getCode)
+        .withField("message", STRING_TYPE, ErrorListBadRequest::getMessage)
+        .withField(
```

Comment:
will be `withOptionalField` I think? im not sure we ever send stacktrace...

Reply:
In the cases it is used we do